descartes_light/solvers/include/descartes_light/solvers/bgl/bgl_dijkstra_solver.h

```diff
@@ -38,6 +38,40 @@ class BGLEfficientDijkstraSVSESolver : public BGLSolverBaseSVSE<FloatType>
 using BGLEfficientDijkstraSVSESolverF = BGLEfficientDijkstraSVSESolver<float>;
 using BGLEfficientDijkstraSVSESolverD = BGLEfficientDijkstraSVSESolver<double>;
 
+/**
+ * @brief BGL solver implementation that constructs vertices build function and uses Dijkstra's
+ * algorithm with an edge-adding visitor to search the graph
+ */
+template <typename FloatType>
+class BGLDijkstraSVDESolver : public BGLSolverBaseSVDE<FloatType>
+{
+public:
+  using BGLSolverBaseSVDE<FloatType>::BGLSolverBaseSVDE;
+
+  SearchResult<FloatType> search() override;
+};
+
+using BGLDijkstraSVDESolverF = BGLDijkstraSVDESolver<float>;
+using BGLDijkstraSVDESolverD = BGLDijkstraSVDESolver<double>;
```

Comment:
Is it valuable to add this intermediate solver? It might be better to just add this event visitor to the `BGLEfficientDijkstraSolver` (and possibly rename it to `BGLDijkstraSVDESolver`) which is intended to represent the "best" Dijkstra solver we have to date.

Reply:
I think they are worthwhile for benchmarking. 
The show direct comparison for static vs. dynamic edge construction, and how this difference is exacerbated by efficient vs. full Dijkstra search.

However, I do not see a scenario where dynamic full dijkstra would be used in reality

Reply:
I have an idea of how we can parameterize these solvers on the visitors that are used in them. I'll address that in a separate PR. This is good for now